hr_employee_exemption/hr_employee_exemption.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding:utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Savoir-faire Linux. All Rights Reserved.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.osv import orm, fields
+
+
+class hr_employee_exemption(orm.Model):
+    _name = 'hr.employee.exemption'
+    _description = 'Employee Income Tax Exemption'
+
+    _columns = {
+        'employee_id': fields.many2one(
+            'hr.employee',
+            'Employee',
+            required=True,
+            ondelete='cascade',
+        ),
+        'exemption_id': fields.many2one(
```

Comment:
I agree that it can be more logical and you avoid the formula in your salary rule.